src/libs/print_settings.c

```diff
@@ -57,7 +57,7 @@ typedef struct dt_lib_print_settings_t
 {
   GtkWidget *profile, *intent, *style, *style_mode, *papers;
   GtkWidget *printers, *orientation, *pprofile, *pintent;
-  GtkWidget *width, *height;
+  GtkWidget *width, *height, *bpc;
```

Comment:
I would call it `black_point_compensation` so it doesn't get confused with `bit per channel`.